Tasks/Gradle/gradletask.ts

```diff
@@ -131,20 +131,25 @@ function configureWrapperScript(wrapperScript: string): string {
 // update JAVA_HOME if user selected specific JDK version or set path manually
 function setJavaHome(javaHomeSelection: string): void {
     let specifiedJavaHome: string;
+    let javaTelemetryData;
 
     if (javaHomeSelection === 'JDKVersion') {
         tl.debug('Using JDK version to find and set JAVA_HOME');
         let jdkVersion: string = tl.getInput('jdkVersion');
         let jdkArchitecture: string = tl.getInput('jdkArchitecture');
-
+        javaTelemetryData = { "jdkVersion": jdkVersion };
```

Comment:
Any reason we don't get the `architecture` at the same time? :)

Reply:
No particular reason, for us Java version is interesting since we want to see how many JavaALM 6 vs 7 vs 8 vs custom path builds are running. Architecture is not something we are targeting right now. For telemetry, I am trying to define scoped scenarios of value and add reports for that instead of trying to get everything.